### DISCUSSION TITLE
Looking for Beta-Testers (Open Beta)

### DISCUSSION THREAD

**Discussion by astubenbord** [astubenbord/paperless-mobile#156] [Announcements]
Hi all, 

the next version is just around the corner, containing major updates like **support for multiple accounts**  and **document bulk edits**.

Before I release it though, I need some users to test the upcoming version since this is kind of a bigger update. So please sign up for the open beta in Google Play once it gets released! :)

I'll post an update here once it's online.

**Comment by rubenixnagios** [comment]
hope to get translations done for this beauty b4 the update! sent u a message tru crowdin!!

**Reply by astubenbord** [reply]
Just created it :) Thanks for contributing!

**Reply by rubenixnagios** [reply]
lovely

**Comment by astubenbord** [comment]
Just submitted it to Google play, should not take that long to be available

**Comment by lagset** [comment]
Hey, I just stumbled upon this project. This app seems really awesome. Great experience when browsing paperless via smartphone.

I thought I give it a try and join the Playstore beta. 

I hope to get to more testing, for now I found the following issue: It appears that filtering by Correspondent and Document Type is broken.

When I navigate into my labels and filter for one specific document type I get a list of the most recent documents. The number of listed documents equals the number of documents this filter (truly) applies to.

E.g. I have a Document Type "Foo" which has 4 documents assigned. This count is  displayed correctly next to the filter action icon of the document type. If I click on the filter action of the document type I am redirected to the view "linked documents" which now incorrectly displays the four most recent documents - not the four correct ones. 

The same applies to the documents view when filtering manually by Correspondent or Document Type.

**Reply by astubenbord** [reply]
Hi, thanks for reporting :) Will be fixed in the next release, there was a typo in the query parameter during the refactoring.

**Reply by astubenbord** [reply]
New version is on the way, release workflow is running, should be up this evening

**Reply by lagset** [reply]
JFYI: I updated to the new beta version. After that the app was only showing the starting screen (paperless logo). Clearing cache and stopping the app didn't work, a reinstall fixed it.

With the new version I got a rather obvious visual issue with the user image in the header:

![Screenshot_20230429-005946-01.jpeg](https://user-images.githubusercontent.com/15156011/235267549-09b16dc0-b1c4-4084-bf12-eaf1ae280ff5.jpeg)

**Reply by astubenbord** [reply]
You mean even after reinstalling?

**Reply by lagset** [reply]
Yes this visual issue is there after reinstall.
When tapping on it the account overview opens, but the layout is distorted as well:

![Screenshot_20230429-094420.png](https://user-images.githubusercontent.com/15156011/235291399-cea6766a-e0c1-4f3d-afe6-f5f04405b451.png)

I am using Pixel 3a (Android 12) and paperless-ngx version 1.14.1

**Reply by astubenbord** [reply]
Okay it seems there is an issue with the user avatar. Not sure I have time to investigate today, but I'll try to fix it as soon as possible.

**Reply by lagset** [reply]
Issues I found so far:

**Saving**
I can not save any changes to documents. Tapping on the _Save Changes_ Buttons simply doesn't do anything (besides visual feedback of the button).

**Uploading: Original filename no full-stop**
Minor Issue: Uploading documents works. But the full-stop before the file ending (tested with PDFs) gets lost. 

E.g. Uploading a file with title "Foo" and Filename "foo-2023-03.pdf" will result in a _Original filename_ "foo-2023-03pdf" on paperless-ngx server.


**Uploading: Unknown error with blank input fields**
Issue: Uploading does only work when **all** Input fields marked with a _*_ are filled with a value. Leaving at least one of them blank results in a unknown error preventing the upload.

However, the help text below the input fields suggests otherwise: "If you specify values for these fields, your paperless instance will not automatically derive a value. If you want these values to be automatically populated by your server, leave the fields blank."

**Reply by astubenbord** [reply]
**Regarding saving:**
Paperless apparently now expects the owner id to be sent in the payload, which was not required before. This will be updated in the upcoming release.

Regarding the upload: I'm not sure what's the problem there, but I'll have a look. I think they now expect the owner id in the request body when saving a document. I'll also look through the changelog and see if they changed the upload endpoint (I think they added quite some fields). 

Anyway, if you find more bugs, please let me know, I appreciate it.
